frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1,63 +1,41 @@
 /* eslint-disable @typescript-eslint/no-use-before-define */
 import {
-  Action,
   Alert,
   AlertSeverity,
   AlertStates,
   BlueInfoCircleIcon,
-  FormatSeriesTitle,
   GreenCheckCircleIcon,
   PrometheusAlert,
-  PrometheusEndpoint,
-  PrometheusLabels,
   RedExclamationCircleIcon,
-  ResourceStatus,
   RowFilter,
   RowProps,
   Rule,
   Silence,
   SilenceStates,
   TableColumn,
-  useResolvedExtensions,
   YellowExclamationTriangleIcon,
 } from '@console/dynamic-plugin-sdk';
 import {
-  AlertingRuleChartExtension,
-  AlertingRulesSourceExtension,
-  isAlertingRuleChart,
-  isAlertingRulesSource,
-} from '@console/dynamic-plugin-sdk/src/extensions/alerts';
-import {
-  ActionServiceProvider,
   ListPageFilter,
-  ResourceLink,
   Timestamp,
   useListPageFilter,
   VirtualizedTable,
 } from '@console/dynamic-plugin-sdk/src/lib-core';
 import { consoleFetchJSON } from '@console/dynamic-plugin-sdk/src/utils/fetch';
 import { withFallback } from '@console/shared/src/components/error';
-import { QueryBrowser } from '@console/shared/src/components/query-browser';
-import { formatPrometheusDuration } from '@openshift-console/plugin-shared/src/datetime/prometheus';
 import { useExactSearch } from '@console/app/src/components/user-preferences/search';
 import {
   Alert as PFAlert,
   Breadcrumb,
   BreadcrumbItem,
+  DropdownItem,
   Button,
   Checkbox,
-  CodeBlock,
-  CodeBlockCode,
   Flex,
   FlexItem,
   Label,
   Modal,
   ModalVariant,
-  Popover,
-  Toolbar,
-  ToolbarContent,
-  ToolbarGroup,
-  ToolbarItem,
 } from '@patternfly/react-core';
 import {
   Dropdown as DropdownDeprecated,
```

Comment:
@logonoff, these deprecated components need to be removed and replaced to fully satisfy https://issues.redhat.com/browse/OU-561.